cp-overrides.yaml

```diff
@@ -1,10 +1,19 @@
 schema_version: 1
 modules:
-      repositories:
-          - name: jboss-eap-modules
-            git:
-                  url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: master
+
+# Example artifact image.yaml overrides:
+# concreate --overrides=<filenaame.yaml>
```

Comment:
Why not
concreate --overrides=cp-overrides.yaml

?

Since these comments are in the specific override files, having them match the exact file is clearer.

Arguing against my own point: I can see that for the expected use case we wouldn't want people who are doing experiments with other artifacts to edit this file (as that might get committed), rather they should create their own file and use it. So this is kind of just the spot for docs.

Reply:
I think we should just document dev-overrides.yaml and leave the other two bare.